can/interfaces/pcan/pcan.py

```diff
@@ -651,11 +597,12 @@ def _detect_available_configs():
     def status_string(self) -> Optional[str]:
         """
         Query the PCAN bus status.
-        :return: The status in string.
+
+        :return: The status description, if any was found.
         """
-        if self.status() in PCAN_DICT_STATUS:
+        try:
```

Comment:
This change prevents a hypothetical error if `self.status()` changes between lines.